src/Tribe/Customizer.php

```diff
@@ -125,7 +125,7 @@ public function __construct() {
 		 * By default inline scripts will be printed on the `wp_enqueue_scripts` action, but other
 		 * plugins or later iterations might require inline styles to be printed on other actions.
 		 *
-		 * @since TBD
+		 * @since 4.12.15
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f749ccc26a087aba4ab6ca5c1c3e3d4d</sub>